ruff.toml

```diff
@@ -10,13 +10,13 @@ exclude = [
 [lint]
 extend-select = [
 	# upstream
- 
+
 	"C901", # complex-structure
 	"I", # isort
 	"PERF401", # manual-list-comprehension
 	"W", # pycodestyle Warning
- 
- 	# Ensure modern type annotation syntax and best practices
+
+	# Ensure modern type annotation syntax and best practices
```

Comment:
These cleanups of whitespace ideally should be done in `skeleton`, otherwise there might be some merge conflicts in the future, isn't it?

Reply:
Those whitespaces are odd. (btw that was autoformatted). Not sure if I accidentally added them in the skeleton or if Jason accidentally introduced them when merging. I'll take a look, but yeah.

Reply:
https://github.com/jaraco/skeleton/pull/164